src/test/java/com/endava/grocy/entity/shoppinglist/TestShoppingList.java

```diff
@@ -0,0 +1,116 @@
+package com.endava.grocy.entity.shoppinglist;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestShoppingList extends TestBaseShoppingList {
```

Comment:
We can now extent the TestBaseClass in order to have access to the seleniumConfig property